h3-quinn/Cargo.toml

```diff
@@ -19,6 +19,6 @@ quinn = { version = "0.10", default-features = false, features = [
     "futures-io",
 ] }
 quinn-proto = { version = "0.10", default-features = false }
-tokio-util = { version = "0.7.7" }
-futures = { version = "0.3.27" }
-tokio = { version = "1.28", features = ["io-util"], default-features = false }
+tokio-util = { version = "0.7.9" }
+futures = { version = "0.3.28" }
+tokio = { version = "1.33", features = ["io-util"], default-features = false }
```

Comment:
Is such a newer version of Tokio required? If not, we can keep the lower number, so it works with older versions. Anyone who had a newer one still works too.

Reply:
Not required i think. I will change that.